uint32 size
uint16 x
uint16 y
uint16 width
uint16 height

uint8 NOT_LOCKED=0
uint8 LOCKED=1

uint8 status

uint8 UB=0
uint8 FACE=1

uint8 type

